src/bflib_vidsurface.c

```diff
@@ -51,7 +51,7 @@ TbResult LbScreenSurfaceCreate(struct SSurface *surf,unsigned long w,unsigned lo
         format = lbDrawSurface->format;
     }
     //SDL_HWSURFACE
-    surf->surf_data = SDL_CreateRGBSurface(SDL_SRCCOLORKEY , w, h, format->BitsPerPixel,
+    surf->surf_data = SDL_CreateRGBSurface(SDL_TRUE , w, h, format->BitsPerPixel,
```

Comment:
![image](https://user-images.githubusercontent.com/68837817/89727897-6a4e7200-da31-11ea-8ceb-bbe8c059ce0b.png)

Reply:
Looks like I confused this with this part from migration guide:
```
Colorkey: When calling SDL_SetColorKey(), you should pass SDL_TRUE instead of SDL_SRCCOLORKEY.
```
Will change that.

Reply:
ok